### PR TITLE
Add `concatAST` utility function

### PR DESCRIPTION
**Pull request by vhenzl**
Cherry-picked from #929.

### CODE REVIEW

File: tests/Utils/ConcatASTTest.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+use GraphQL\Utils\AST;
+use PHPUnit\Framework\TestCase;
+
+use function preg_match;
+use function preg_replace;
+use function trim;
+
+class ConcatASTTest extends TestCase
+{
+    private function dedent(string $str): string
```
Comment (spawnia) [inline tests/Utils/ConcatASTTest.php:18]: Is this going to be needed for multiple tests?

Comment (vhenzl) [reply tests/Utils/ConcatASTTest.php:18]: Yeah, definitely. I thought I would later extract it somewhere. But I'm not quite sure where to put it. `graphql-js` has [`__testUtils__`](https://github.com/graphql/graphql-js/tree/main/src/__testUtils__) in `src`. Maybe we could colocate testing-related stuff to `tests/TestUtils`?

I'm also wondering if make it a static method like `Dedent::dedent()` (unnecessary duplication) or `Utils::dedent()` (yet another `Utils` class), or if wouldn't make more sense to have it just like a function (in a namespace).

Comment (spawnia) [reply tests/Utils/ConcatASTTest.php:18]: `Tests\TestUtils::dedent()` seems good.

Comment (vhenzl) [reply tests/Utils/ConcatASTTest.php:18]: I was thinking more about `TestUtils` directory and putting there also the "kitchen sink" and other things scattered elsewhere. But we can look at that later…

And I have just noticed, there is `TestUtils` class in `test/Language`. Should I move it to the root, or create another class?

Comment (spawnia) [reply tests/Utils/ConcatASTTest.php:18]: Do what you think is right, I will review it.

Comment (spawnia) [reply tests/Utils/ConcatASTTest.php:18]: Can you update this with nowdoc syntax?

**Comment by vhenzl** [comment]
This should be good to go now.

**Pull request merged**